tidb-in-kubernetes/google-kubernetes-tutorial.md

```diff
@@ -0,0 +1,200 @@
+
+To connect to TiDB within the Kubernetes cluster, you can establish a tunnel between the TiDB service and your Cloud Shell. This is recommended only for debugging purposes, because the tunnel will not automatically be transferred if your Cloud Shell restarts. To establish a tunnel:
+
+	kubectl -n tidb port-forward svc/demo-tidb 4000:4000 &>/tmp/port-forward.log &
+
+From your Cloud Shell:
+
+	sudo apt-get install -y mysql-client &&
+	mysql -h 127.0.0.1 -u root -P 4000
+
+Try out a MySQL command inside your MySQL terminal:
+
+	select tidb_version();
+
+If you did not specify a password in helm, set one now:
+
+	SET PASSWORD FOR 'root'@'%' = '<change-to-your-password>';
+
+_Note that, this command contains some special characters which cannot be
+auto-populated in the google cloud shell tutorial: you need to copy and paste it into your console manually._
+
+Congratulations, you are now up and running with a distributed TiDB database compatible with MySQL!
+
+## Scale out the TiDB cluster
+
+With a single command we can easily scale out the TiDB cluster. To scale out TiKV:
+
+	helm upgrade demo pingcap/tidb-cluster --set pd.storageClassName=pd-ssd,tikv.storageClassName=pd-ssd,tikv.replicas=5 --version=${chartVersion}
+
+Now the number of TiKV pods is increased from the default 3 to 5. You can check it with:
+
+	kubectl get po -n tidb
+
+## Accessing the Grafana dashboard
+
+To access the Grafana dashboards, you can create a tunnel between the Grafana service and your shell.
+To do so, use the following command:
+
+    kubectl -n tidb port-forward svc/demo-grafana 3000:3000 &>/dev/null &
+
+In Cloud Shell, click on the Web Preview button and enter 3000 for the port. This will open a new browser tab pointing to the Grafana dashboards. Alternatively, use the following URL https://ssh.cloud.google.com/devshell/proxy?port=3000 in a new tab or window.
+
+If not using Cloud Shell, point a browser to `localhost:3000`.
+
+## Destroy the TiDB cluster
+
+When the TiDB cluster is not needed, you can delete it with the following command:
+
+	helm delete demo --purge
+
+The above commands only delete the running pods, the data is persistent. If you do not need the data anymore, you should run the following commands to clean the data and the dynamically created persistent disks:
+
+	kubectl delete pvc -n tidb -l app.kubernetes.io/instance=demo,app.kubernetes.io/managed-by=tidb-operator &&
+	kubectl get pv -l app.kubernetes.io/namespace=tidb,app.kubernetes.io/managed-by=tidb-operator,app.kubernetes.io/instance=demo -o name | xargs -I {} kubectl patch {} -p '{"spec":{"persistentVolumeReclaimPolicy":"Delete"}}'
+
+## Shut down the Kubernetes cluster
+
+Once you have finished experimenting, you can delete the Kubernetes cluster with:
+
+	gcloud container clusters delete tidb
+
+
+## More information
+
+For production deployments, view our [operation guide](./operation-guide.md), and look at the GKE section.
```

Comment:
```suggestion
If you want to deploy TiDB on GCP GKE in the production environment, you must read the GKE section in our [operation guide](./operation-guide.md) first.
```
Change this link into an absolute link. And make sure it works before merging this PR.